test/singlestoredb.test.ts

```diff
@@ -0,0 +1,43 @@
+import { format as originalFormat, FormatFn } from 'src/sqlFormatter';
+import behavesLikeSqlFormatter from './behavesLikeSqlFormatter';
+
+import supportsAlterTable from './features/alterTable';
+import supportsBetween from './features/between';
+import supportsComments from './features/comments';
+import supportsConstraints from './features/constraints';
+import supportsCreateView from './features/createView';
+import supportsDeleteFrom from './features/deleteFrom';
+import supportsDropTable from './features/dropTable';
+import supportsIdentifiers from './features/identifiers';
+import supportsInsertInto from './features/insertInto';
+import supportsParams from './options/param';
+import supportsStrings from './features/strings';
+import supportsTruncateTable from './features/truncateTable';
+import supportsUpdate from './features/update';
+
+describe('SingleStoreDbFormatter', () => {
+  const language = 'singlestoredb';
+  const format: FormatFn = (query, cfg = {}) => originalFormat(query, { ...cfg, language });
+
+  // behavesLikeMariaDbFormatter(format);
+  behavesLikeSqlFormatter(format);
+  supportsComments(format, { hashComments: true });
+  supportsStrings(format, ["''", '""', "X''"]);
+  supportsIdentifiers(format, ['``']);
+  supportsCreateView(format, { orReplace: false });
+  supportsDropTable(format, { ifExists: true });
+  supportsConstraints(format);
+  supportsAlterTable(format, {
+    addColumn: true,
+    dropColumn: true,
+    modify: true,
+    renameTo: true,
+    renameColumn: false,
+  });
+  supportsDeleteFrom(format);
+  supportsInsertInto(format, { withoutInto: true });
+  supportsUpdate(format);
+  supportsTruncateTable(format, { withoutTable: true });
+  supportsBetween(format);
+  supportsParams(format, { positional: false });
```

Comment:
Really there's no point of calling `supportsParams()` in this test at all.

Reply:
Should add more documentation on _why_ this is the case for outsiders building new tests since it may not be as intuitive if they haven't been working with the code as much as we have